### PR TITLE
Added example with custom seeker colors and fixed animations example

### PR DESCRIPTION
**Pull request by balavor**
## Changes 
- [x] Added example with custom colors of the seeker;
- [x] Fixed incorrect behavior of example with disabled animation;
- [x] Updated Readme.

## Example
screen shot 2018-08-22 at 2 52 02 pm 

@aol-public/mobile-sdk-team Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,6 +291,14 @@ We've added some new animations to our default player controls UX. If you want t
 
 > [Animations Customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
 
+#### Seeker Custom Colors
```
Comment (AndriiMoskvin) [inline README.md]: Please, update the table of contents also.

File: README.md [hunk 2]
```diff
@@ -291,6 +291,14 @@ We've added some new animations to our default player controls UX. If you want t
...
+
+Latest update brought in our SDK an ability to set up custom colors for all seeker elements, including the current time, cue points, progress bar and all others. You can also set up a custom color only for a single element when the others colors will be the same as tint color.
+
+##### _Tutorial Sample:_
+
+> [Animations Customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
```
Comment (AndriiMoskvin) [inline README.md]: Why `Animations Customization`?

**Review by AndriiMoskvin — DISMISSED** [review]

**Review by AndriiMoskvin — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -290,7 +291,15 @@ We've added some new animations to our default player controls UX. If you want t
 
 ##### _Tutorial Sample:_
 
-> [Animations Customization](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
+> [Animations Customization and Disabling](https://github.com/aol-public/OneMobileSDK-playground-ios/tree/master/NewTutorials)
+
+#### Custom Colors for Seeker Elements
+
+The latest update includes a new feature to set custom colors for each element of player's seeker, like current time, progress, cue points etc. It is also possible to set custom color only for one of seeker elements, when the others will have the same color as the view's tint color.
```
Comment (AndriiMoskvin) [inline README.md]: Reference which one update, please.

**Review by AndriiMoskvin — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AndriiMoskvin) [reply README.md]: @RomanTysiachnik ☝️

**Review by AndriiMoskvin — DISMISSED** [review]

**Comment by AndriiMoskvin** [comment]
Waiting for SDK release 2.28. Don't merge.

**Comment by MarkGerl** [comment]
Looks good to me @RomanTysiachnik

**Comment by balavor** [comment]
@AndriiMoskvin waiting for your review.

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**